docs/docs/guides/python-file-upload.md

```diff
@@ -0,0 +1,42 @@
+# Python File Upload
+
+```python
+#!/usr/bin/python3
+
+import requests
+import os
+from datetime import datetime
+
+API_KEY = 'YOUR_API_KEY'                # replace with a valid api key
+BASE_URL = 'http://127.0.0.1:2283/api'  # replace as needed
+
+
+def upload(file):
+    stats = os.stat(file)
+
+    headers = {
+        'Accept': 'application/json',
+        'x-api-key': API_KEY
+    }
+
+    data = {
+        'deviceAssetId': f'{file}-{stats.st_mtime}',
+        'deviceId': 'python',
```

Comment:
Do we still require these?

Reply:
oh yeah

Reply:
they aren't even nullable in the database

Reply:
gotta tell those mobile developers to stop using them already